modules/aws/eks/outputs.tf

```diff
@@ -49,3 +49,10 @@ output "prometheus_config" {
     role_arn = module.prometheus.role_arn
   }
 }
+
+output "starboard_config" {
+  description = "Configuration for Starboard"
+  value = {
+    role_arn = module.starboard_ecr["starboard"].role_arn
+  }
+}
```

Comment:
Can I have a output that got a if statment and check if starboard is enabled?
Isn't the output always going to run no matter what or will it just be a nul value?